src/github-api.ts

```diff
@@ -98,11 +98,17 @@ export async function fetchOtherRunStatus(
   octokit: Parameters<typeof fetchRunSummaries>[0],
   params: Parameters<typeof fetchRunSummaries>[1],
   ownJobIDs: Readonly<Set<JobID>>,
+  jobsToWaitFor: string[],
 ): Promise<Report> {
   const checkRunSummaries = await fetchRunSummaries(octokit, params);
-  const otherRelatedRuns = checkRunSummaries.flatMap<CheckRunsSummary>((summary) =>
+  const filteredCheckRunSummaries = jobsToWaitFor.length === 0
+    ? checkRunSummaries
+    : checkRunSummaries.filter(summary => jobsToWaitFor.includes(summary.source.name));
```

Comment:
Using `Array.prototype.includes()` is a loop in a loop. So please prefer [Set.prototype.has()](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set/has) as 
 https://github.com/kachick/wait-other-jobs/blob/bc780bff767b2241e469c48f7abafa59f1c5addc/src/github-api.ts#L104